content/en/blog/_posts/2024-03-12-mid-cycle-1.30.md

```diff
@@ -103,7 +103,7 @@ docs](/docs/reference/access-authn-authz/authorization/#configuring-the-api-serv
 Horizontal pod autoscaling based on `ContainerResource` metrics will graduate to stable in v1.30.
 This new behavior for HorizontalPodAutoscaler allows you to configure automatic scaling based on the
 resource usage for individual containers, rather than the aggregate resource use over a Pod. See our
-[previous article](2023/05/02/hpa-container-resource-metric/) for further details, or read
```

Comment:
https://kubernetes.io/blog/2024/03/12/kubernetes-1-30-upcoming-changes/#container-resource-based-pod-autoscaling-kep-1610-https-kep-k8s-io-1610

Reply:
Thanks @pacoxu
The preview looks good to me.
https://deploy-preview-45517--kubernetes-io-main-staging.netlify.app/blog/2024/03/12/kubernetes-1-30-upcoming-changes/#container-resource-based-pod-autoscaling-kep-1610-https-kep-k8s-io-1610

/lgtm